### ISSUE TITLE
New section "Security hardening"

### ISSUE THREAD

**Issue by franzliedke** [flarum/docs#15]
As proposed by @Kulga [here](https://github.com/flarum/flarum/commit/569e3a3b5ec862ecae93e0e438dd6a7a4c5012ed#commitcomment-21080215), we should probably change our included Apache configuration (and the one for Nginx in the docs) to only include the bare essentials - or at least only things that cause no problems with other components.

(In this particular example, always stripping the `Proxy` request header might cause problems with reverse proxies.)

Instead, a section called "Security hardening" or something similar should contain further "recipes" with things that should be applied to secure the server configuration. (For things like the above, notes should make it clear that they should not be combined with certain setups, e.g. reverse proxies.)

**Comment by franzliedke** [comment]
Indeed, with the move to the public folder, a bunch of rules can be removed from our .htaccess file (and the corresponding Nginx config example in the docs).

Example: https://github.com/flarum/flarum/blob/0e0d443f85f0e88286e5451d0bdba4a6e0f9e747/public/.htaccess#L26-L30

**Comment by imorland** [comment]
Closing as stale — this issue is many years old (predates the 2.x docs and in most cases the current docs structure entirely). The docs have been substantially rewritten since. If this is still a real gap on the current site, please open a fresh issue with up-to-date detail.
